### DISCUSSION TITLE
Default URL change since update from 3.9.? to 3.11.2

### DISCUSSION THREAD

**Discussion by lheureuxbenoit** [code4recovery/12-step-meeting-list#455] [Q&A]
Hello, 

I did an update and the default url meetings.aa87.org/meetings is not working. 

I can go to /wp-admin. If I replace the meetings by reunions, it is working. 
https://meetings.aa87.org/reunions/

I understand that the new version in french change the default url for reunion. I supposed I should reinstall the extension but is there a way to change that default URL?

thank you

Submit by Benoit L’Heureux
admin@aa87.org

[2021-07-15 aa87 meetings list error.pdf](https://github.com/code4recovery/12-step-meeting-list/files/6822778/2021-07-15.aa87.meetings.list.error.pdf)

**Comment by tech2serve** [comment]
I suspect one of two things is going on here:

1. You or someone added at override for the default list location. Please see [this FAQ entry](https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#can-i-change-the-url-of-the-meetings-list). If the modification is present, you may need to roll it back to the default/remove the mod. Possibly in a child theme?
2. Regardless, given the changes stemming from your upgrade, you may need to click on Settings --> Permalinks, and then click Save. This should refresh all the links.

Please search our Discussions here for "permalinks" for more information.

Let us know if this solves things, please.

**Comment by lheureuxbenoit** [comment]
This WP web site is dedicate to 12SML. 
There is no child and there is no modification. 
Before submitting the request, I did: Settings --> Permalinks, and then click Save

Admin mode, under Settings/General/WordPress Address (URL) =  https://meetings.aa87.org: this redirect to https://meetings.aa87.org/meetings that is not working. SInce the update, the URL to get the meetings is https://meetings.aa87.org/reunions.

How to change the WordPress Address (URL)  p;ointing to /reunions?

**Reply by tech2serve** [reply]
So this is not a TSML issue. I have not personally tweaked the Wordpress Address (URL) field.

It appears that someone set the redirection (through a VHOST?, possibly DNS?) for "meetings.aa87.org" to the "/meetings' URL, and that redirection is what you need to update. I don't think it will matter what you do in the plugin as the pointer is hard coded. 

I noticed that if I navigate to aa87.org, and then click on the meetings lists, it works correctly.

You'll need to change that redirection to "https://meetings.aa87.org/reunions". 

I do not believe I can be of further help as this is most likely control or configured at your hosting provider or on your VPS if that is what you're using. 

Good luck!

**Comment by lheureuxbenoit** [comment]
Ok, last chance!

When I instal TSML, the WP url is the domain name like: meetings.aa87.org
and this URL redirect automatically to meetings.aa87.org/meetings.
And it is the same if I set the WP to EN or FR. 

Since the update: 
The redirection by default modify by an TSML to /meetings is not working because the default url is now /reunions

If I change the language to English, then by default TSML is working with mydomain.com/meetings
But we want to set to FR. 

This is a TSML problem!

I have a feeling that if we reinstall a new TSML and reimport wverything, it will work fine with with /reunions.

But not very funny to do just for an update.

;-/

**Reply by joshreisner** [reply]
Hey @lheureuxbenoit ,

Do you have access to your theme?

Try adding `$tsml_slug = 'meetings';` somewhere in your theme's functions.php file. This should return the URL to `/meetings`

**Comment by tech2serve** [comment]
I engaged with a couple of our members who are smarter than I am, especially regarding translations. One did find a line of code that was changed about a year ago which *may* be leading to the issue. We'll look at that closer.

In the meantime, the consensus is that you should be able to use the information I provided above to make a customization. Specifically, [the entry in our FAQ](https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#can-i-change-the-url-of-the-meetings-list) about setting `$tsml_slug = 'meetings'`.

Try adding that towards the top (but after `<?php`) of your `functions.php` file. Without a child theme, it will be overwritten next time you update, but at least this should enable to you to test and make sure this will work for you.

Please let us know.
